File: football-ticket-service/src/main/java/com/app/footballticketservice/dto/TicketDTO.java
package com.app.footballticketservice.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class TicketDTO {
    private Long typeId;
    private String name;
    private Long matchId;
    private BigDecimal price;
    private String description;
    private Long seatId;
    private String seatNumber;
}

File: football-ticket-ui/src/app/share/model/TicketDTO.ts
import {SeatDTO} from './SeatDTO';

export class TicketDTO {
  constructor(
    public typeId: number = 0,
    public name: string = '',
    public matchId: number = 0,
    public price: number = 0,
    public description: string = '',
    public seats: SeatDTO[] = [],
  ) {
  }
}
